src/components/Utils/normalize.js

```diff
@@ -0,0 +1,7 @@
+const normalize = (value) => {
```

Comment:
normalize doesn't say anything, please give more self-explanatory name

Reply:
right now it looks like `normalizeEmptySelect` to me, but it could be general `normilizeSelectValue` or something

Reply:
Basically, if we are going to use it in all selects, it might be a good idea to create HOC like OrdersSelect and use it across Orders app